main.go

```diff
@@ -6,9 +6,9 @@ import (
 )
 
 var (
-	version = "0.1.1"
-	commit  = "b344608"
-	date    = "20.04.2022"
+	version = "0.1.2"
```

Comment:
Personally, I thing this merits a 0.2.0. 

Also the Date needs adjustment.

Reply:
You are right, this PR adds an "new" functionality, so the minor version has to be raised.

The date ist already adjusted

Reply:
The date is still "25.04.2022"